buildah.go

```diff
@@ -28,7 +28,7 @@ const (
 	Package = "buildah"
 	// Version for the Package.  Bump version in contrib/rpm/buildah.spec
 	// too.
-	Version = "1.19.2"
+	Version = "1.19.3"
```

Comment:
you need to bump the contrib/rpm/buildah.spec too, and it would be good to update changelog.txt and CHANGELOG.md while you're at it.

Reply:
Ok I jumped the gun on this release, since there is at least one more thing we want to back port.  Removing the release and only adding the cherry-pick